doc/docs/index.md

```diff
@@ -4,12 +4,27 @@ nRF Connect Programmer is an app available from [nRF Connect for Desktop](https:
 
-### Deprecated support
-
-- nRF51 Series DKs and Dongle
+|                     | PCA number | Programming with J-Link  | Programming with Nordic Secure DFU  | Programming with MCUboot |
+|---------------------|------------|--------------------------|-------------------------------------|--------------------------|
+| nRF9161 SiP         | PCA10153   | Yes                      | No                                  | No                       |
+| nRF9160 SiP         | PCA10090   | Yes                      | No                                  | No                       |
+| nRF9131 SiP         | PCA10147   | Yes                      | No                                  | No                       |
+| Nordic Thingy:91    | PCA20035   | Yes                      | No                                  | Yes                      |
+| nRF7002 DK          | PCA10143   | Yes                      | No                                  | No                       |
+| nRF5340 DK          | PCA10095   | Yes                      | No                                  | No                       |
+| nRF5340 Audio DK    | PCA10121   | Yes                      | No                                  | No                       |
+| Nordic Thingy:53    | PCA20053   | Yes                      | No                                  | Yes                      |
+| nRF52840 DK         | PCA10056   | Yes                      | No                                  | No                       |
+| nRF52840 Dongle     | PCA10059   | No                       | Yes                                 | No                       |
+| nRF52833 DK         | PCA10100   | Yes                      | No                                  | No                       |
+| nRF52832 DK (nRF52) | PCA10040   | Yes                      | No                                  | No                       |
+| nRF52820 DK         | PCA10115   | No                       | No                                  | No                       |
+| nRF52811 DK         | PCA10068   | No                       | No                                  | No                       |
+| Nordic Thingy:52    | PCA20020   | No                       | No                                  | No                       |
+| nPM1300 EK          | PCA10152   | Yes                      | No                                  | Yes                      |
+| nPM1100 EK          | PCA10123   | Yes                      | No                                  | No                       |
+| nRF51 DK            | PCA10028   | No                       | No                                  | No                       |
+| nRF51 Dongle        | PCA10031   | No                       | No                                  | No                       |
+| Custom kits         | n/a        | Yes for all that feature Nordic SoCs  | No                     | No                       |
```

Comment:
yes for all Nordic SOCs that nrfutil supports so for example nRF52811-QFAAAA is not supported by nrfutil